### PR TITLE
Adding first pass at Ceph docs

### PR DESCRIPTION
**Pull request by hyperbolic2346**

### CODE REVIEW

**Comment by evilnick** [comment]
Cool. Do you want me to suggest changes in the PR or submit fixes to your branch?
It's completely up to you, it's about the same effort to me

**Comment by hyperbolic2346** [comment]
I'm happy with either one. Whatever is easiest for you. We might want to merge and then edit? I don't think this is complete and I think it has too much of a blog feel and not enough of a doc feel. I just think a doc should be more format.

**Comment by evilnick** [comment]
Yes ;)

Okay, I will make edits to your branch then.

File: pages/k8s/ceph.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  unit: ceph-mon/0
+```
+
+## Verifying things are working
+
+Now you can look at the CDK related pieces to verify things are working well:
+
+```bash
+kubectl get sc,po
+NAME                                             PROVISIONER     AGE
```
Comment (evilnick) [inline pages/k8s/ceph.md]: `kubectl get storageclass`  doesn't return any entries for me :(
I feel like there is a step missing

File: pages/k8s/ceph.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+```bash
+$ juju deploy -n 3 ceph-mon
+$ juju deploy -n 3 cs:ceph-osd --storage osd-devices=ebs,32G,2 --storage osd-journals=ebs,8G,1
+```
+
+In this example, we are installing to AWS, using 3 machines for the Ceph monitor, using 3 machines
+for the storage(osd), creating 2 32 gig storage devices per osd machine, and 1 8 gig journal device
+per osd machine. This means overall this cluster with have 192 gigs of storage and 24 gigs of journal
+space.
```
[the remark this answers is not in the document]
Comment (hyperbolic2346) [reply pages/k8s/ceph.md]: Correct, fixing that.

File: pages/k8s/ceph.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+$ juju relate ceph-mon kubernetes-master
+```
+
+Note that the Ceph CSI containers require privileged access:
+
+```bash
+$ juju config kubernetes-master allow-privileged=true
+```
+
+And finally, you need the pools that are defined in the storage class:
```
[the remark this answers is not in the document]
Comment (hyperbolic2346) [reply pages/k8s/ceph.md]: Until CDK charms have the ceph changes I just made, nothing useful will happen. In the future, the templates for the storage actually comes out of cdk-addons, but they will automagically appear when you relate ceph to them. Unfortunately, I don't see a build of kubernetes-worker with the ceph changes yet.

**Comment by hyperbolic2346** [comment]
Thanks for the edits, @evilnick. Looks great!

**Pull request merged**